td-layout/src/runtime.rs

```diff
@@ -38,6 +40,7 @@
 
 pub const TD_PAYLOAD_EVENT_LOG_SIZE: u32 = 0x100000;
 pub const TD_PAYLOAD_ACPI_SIZE: u32 = 0x100000;
+pub const TD_PAYLOAD_MAILBOX_SIZE: u32 = 0x100000;
```

Comment:
Why does it need 1MB? What's the minimal size for this area?

Reply:
Setting this area to 1MB is to reserve space for the AP function since it can't be allocated dynamically.
But at present, it seems that it only consumes about 100 bytes. So I think the minimal size is 4KB (mailbox page) plus size of AP function (about 100 bytes) and the proper one can be 8KB (0x2000).
What do you think of this?

Reply:
I think 2 pages should be ok, one for data and one for code.